securedrop/models.py

```diff
@@ -670,10 +670,10 @@ def login(cls, username, password, token):
         return user
 
     def generate_api_token(self, expiration):
-        # type: (int) -> unicode
+        # type: (int) -> str
         s = TimedJSONWebSignatureSerializer(
             current_app.config['SECRET_KEY'], expires_in=expiration)
-        return s.dumps({'id': self.id}).decode('ascii')
+        return s.dumps({'id': self.id}).decode('ascii')  # type:ignore
```

Comment:
I was concerned this was a bug on our side but it looks like the type annotations for `itsdangerous` just need to be updated upstream: https://github.com/python/typeshed/issues/3081

Reply:
(now fixed upstream, next mypy release we should be able to remove `# type: ignore`, but until then needs to be left in)